### ISSUE TITLE
Stacktrace de-obfuscation is not working

### ISSUE THREAD

**Issue by lucas404x** [getsentry/sentry-dart-plugin#184]
Hi there,

I don't know if it's intentional, but the stacktrace is still obsfucated as you guys can see at the image below:
![image](https://github.com/getsentry/sentry-dart-plugin/assets/42918677/5e8ebd25-b160-4d83-aa7d-ee639493d44f)

This is my current setup in pubspec:
```yaml
sentry:
  upload_debug_symbols: true
  upload_source_maps: true
  upload_sources: false
  project: prodigy-app
  org: technoserv
  # It's not need specify auth token.
  # The sentry_dart_plugin retrieve the AuthToken from SENTRY_AUTH_TOKEN env variable automatically.
  # auth_token: ...
  wait_for_processing: true
  log_level: error # possible values: trace, debug, info, warn, error
  ignore_missing: true
```
And this is the sequence of commands I'm using to build my app:
```powershell
flutter clean
flutter pub get
dart run build_runner build --delete-conflicting-outputs
flutter build apk --flavor $flavor --dart-define-from-file=$envFile --obfuscate --split-debug-info=$buildAppPath/symbols
if ($enableSentry) {
    dart run sentry_dart_plugin
}
```
Is this the expected behavior or am I doing something wrong?

**Comment by buenaflor** [comment]
Hi! Does this affect ios and android?

**Comment by lucas404x** [comment]
My app running on Android only

**Comment by lucas404x** [comment]
One thing that is worth mentioning is everything runs in a Github Action

**Comment by buenaflor** [comment]
Hi, can you send me the event to giancarlo.buenaflor@sentry.io so I can have a look? I can't reproduce it on my end.

**Comment by buenaflor** [comment]
I took a look at your event and it is related to my comment [here](https://github.com/getsentry/sentry-dart/issues/1782#issuecomment-1853774211)

these errors were captured by our error [integration](https://github.com/getsentry/sentry-dart/blob/main/flutter/lib/src/integrations/flutter_error_integration.dart) which sends unhandled errors to Sentry. (that includes 3rd party packages, the flutter framework etc...)

The stacktrace is symbolicated so this is a different issue. I've got some alternatives proposed in my linked comment but we are investigating more to see if there's something better we can do here.
